### PR TITLE
Ubuntu18 image based on Ubuntu 16 debs

### PR DESCRIPTION
**Pull request by Echsecutor**
Changed Ubuntu18 image to use Ubuntu 16 debs.

fixes https://github.com/IDunion/indy-node-container/issues/60

```
# apt policy libsodium18
libsodium18:
  Installed: 1.0.8-5
  Candidate: 1.0.8-5
  Version table:
 *** 1.0.8-5 500
        500 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
        100 /var/lib/dpkg/status
```

### CODE REVIEW

File: build/Dockerfile.ubuntu18 [hunk 1]
```diff
@@ -2,74 +2,65 @@
...
+    dh-python=2.20151103ubuntu1.2 \
+    #
+    && apt-get autoremove -y
 
-COPY --from=builder /usr/local/bin/ /usr/local/bin/
-COPY --from=builder /usr/local/lib/ /usr/local/lib/
-COPY --from=builder /usr/lib/libindy* /usr/lib/
-COPY --from=builder /usr/lib/x86_64-linux-gnu/libssl* /usr/lib/x86_64-linux-gnu/libcrypto* /usr/lib/x86_64-linux-gnu/
 
-RUN useradd -ms /bin/bash indy
```
Comment (mgmgwi) [inline build/Dockerfile.ubuntu18:72]: Why did you delete this line?

Comment (Echsecutor) [reply build/Dockerfile.ubuntu18:72]: the indy user is created by installing the indy node deb package. Trying to useradd him again leads to an error

Comment (mgmgwi) [reply build/Dockerfile.ubuntu18:72]: Ahh, make sense. Thanks.

**Review by mgmgwi — APPROVED** [review]
